cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -177,11 +177,17 @@ class VideosHandlerTestCase(VideoUploadTestMixin, CourseTestCase):
     def test_get_json(self):
         response = self.client.get_json(self.url)
         self.assertEqual(response.status_code, 200)
-        response_videos = json.loads(response.content)["videos"]
+        response_videos_data = json.loads(response.content)
+        response_videos = response_videos_data["results"]
         self.assertEqual(len(response_videos), len(self.previous_uploads))
+        self.assertEqual(response_videos_data["sort_dir"], "asc")
+        self.assertEqual(response_videos_data["total_count"], 20)
+        self.assertEqual(response_videos_data["current_page"], 1)
+        self.assertEqual(response_videos_data["total_pages"], 1)
+        self.assertEqual(response_videos_data["page_size"], 20)
+        self.assertEqual(response_videos_data["sort_field"], "created")
         for i, response_video in enumerate(response_videos):
-            # Videos should be returned by creation date descending
-            original_video = self.previous_uploads[-(i + 1)]
+            original_video = self.previous_uploads[i]
```

Comment:
@shahbazn Can we verify that the videos are in ordered by "created" in asc order. I know we've checked the value in response data but that's not sufficient.